x-pack/test/page_load_metrics/config.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+
+import { FtrConfigProviderContext } from '@kbn/test/types/ftr';
+import { PuppeteerTestRunner } from './runner';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const kibanaCommonTestsConfig = await readConfigFile(
+    require.resolve('../../../test/common/config.js')
+  );
+  const xpackFunctionalTestsConfig = await readConfigFile(
+    require.resolve('../functional/config.js')
+  );
+
+  return {
+    ...kibanaCommonTestsConfig.getAll(),
+
+    testRunner: PuppeteerTestRunner,
+
+    esArchiver: {
+      directory: resolve(__dirname, 'es_archives'),
+    },
+
+    esTestCluster: {
+      ...xpackFunctionalTestsConfig.get('esTestCluster'),
+      serverArgs: [...xpackFunctionalTestsConfig.get('esTestCluster.serverArgs')],
```

Comment:
is it required? you already spread the config `...xpackFunctionalTestsConfig.get('esTestCluster'),`